.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,22 +1,37 @@
+## Overview
 <!--
-  - Thanks for submitting code to Oppia! Please fill out the following as part of
-  - your pull request so we can review your code more easily.
-  -->
+READ ME FIRST:
+Please answer *both* questions below and check off every point from the Essential Checklist!
+If there is no corresponding issue number, fill in N/A where it says [fill_in_number_here] below in point 1.
+-->
 
-## Explanation
-<!--
-  - Explain what your PR does. If this PR fixes an existing bug, please include
-  - "Fixes #bugnum:" in the explanation so that GitHub can auto-close the issue
-  - when this PR is merged.
-  -->
+1. This PR fixes or fixes part of #[fill_in_number_here].
+2. This PR does the following: [Explain here what your PR does and why]
```

Comment:
@seanlip / @rt4914 I'm actually not keen on this. As someone who writes long & detailed descriptions, this makes me feel like I need to write shorter/more terse description which I don't actually think we want to encourage.

Why not keep the current Explanation section as-is? Point 1 is nice since it encourages people to mark PRs as fixing issues, but I rarely see this as an actual problem (people seem to pick up on it quickly), and it can be a nuisance for cases when we really don't have an issue to resolve (such as my recent codeowners PR). Plus, the checklist below already covers this requirement.

Reply:
That's interesting -- the lack of "Fix #X" was a recurring problem on Web. I'm surprised to hear that it's not an issue on Android, but I defer to you on this and additionally agree that this part is not needed given the checkbox.

Given that, keeping the existing Explanation section seems reasonable to me. (Sorry @rt4914 for the back-and-forth on this.)

Reply:
Done.